uahf-technical-spec.md

```diff
@@ -157,43 +157,51 @@ with a specific magic data value of
 without any terminating null character) shall be considered invalid until
 block 530,000 inclusive.
 
-RATIONALE: To give users on the legacy chain (or other fork chains)
+RATIONALE: (DEPRECATED - see NOTE 2) To give users on the legacy chain (or other fork chains)
 an opt-in way to exclude their transactions from processing on the UAHF
 fork chain. The sunset clause block height is calculated as approximately
 1 year after currently planned UASF activation time (Aug 1 2017 00:00:00 GMT),
 rounded down to a human friendly number.
 
-NOTE: Transactions with such OP_RETURNs shall be considered valid again
+NOTE 1: Transactions with such OP_RETURNs shall be considered valid again
 for block 530,001 and onwards.
 
+NOTE 2: With the changes in v1.6 of this specification, mandatory use
+of SIGHASH_FORKID replay protection on UAHF chain makes the use of this
+opt-out protection unnecessary. Clients should nevertheless implement this
+requirement, as removing it would constitute a hard fork vis-a-vis the
+existing network. The sunset clause in this requirement will take care
+of its expiry by itself.
 
-### REQ-6-2 (opt-in signature shift via hash type)
 
-Once the fork has activated, a transaction shall not be deemed invalid if
-the following are true in combination:
-- the nHashType has bit 6 set (mask 0x40)
-- adding a magic 'fork id' value to the nHashType before the hash is
-  calculated allows a successful signature verification as per REQ-6-3
+### REQ-6-2 (mandatory signature shift via hash type)
 
-RATIONALE: To give users on the UAHF chain an opt-in way to encumber
-replay of their transactions to the legacy chain (and other forks which may
-consider such transactions invalid).
+Once the fork has activated, a transaction shall be deemed invalid if
+its nHashType does not have bit 6 set (SIGHASH_FORKID, mask 0x40) and it
+is not using the new digest algorithm described in REQ-6-3.
```

Comment:
I think the removed paragraph is clearer so maybe it should be tweaked instead of rewritten to express this is mandatory now ?